<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>start_page</class>
 <widget class="QWidget" name="start_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>663</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>753</width>
    <height>663</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>663</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EASY AUDIT - HOME PAGE</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="app_logo_hlayout">
     <item>
      <spacer name="hspacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="app_logo_lbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>218</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>218</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>NAME-with-TAGLINE.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vpsacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btn_bar">
     <property name="spacing">
      <number>100</number>
     </property>
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="cis_benchmark_btn">
       <property name="text">
        <string>CIS BENCHMARKS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="new_audit_btn">
       <property name="text">
        <string>NEW AUDIT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="audit_history_btn">
       <property name="text">
        <string>AUDIT HISTORY</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vspacer_1">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="team_name_hlayout">
     <item>
      <spacer name="hspacer_1">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignmentFlag::Qt::AlignmentFlag::AlignBottom">
      <widget class="QLabel" name="team_name_lbl">
       <property name="text">
        <string>@PURSUIT OF PERFECTION</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
